src/main/java/org/usfirst/frc/team1806/robot/subsystems/LiftSubsystem.java

```diff
@@ -148,22 +190,14 @@ public void onLoop(double timestamp) {
 							holdPosition();
 						}
 					}
+					liftStateLoop();
 
-					cubePositionLoop();
-					cubeLiftStateLoop();
+					lastTimeStamp = timestamp;
 				}
             }
 
-            private void cubePositionLoop(){
-            	switch (mLiftPosition){
-					case BOTTOM_LIMIT:
-						return;
-					case TOP_LIMIT:
-						return;
```

Comment:
How is switch, case, return different from switch, case, break?

Reply:
return will end the entire function you're in, and gives you the option to return a value at that point, break just gets you out of the lowest switch, while loop, if block, etc. that the break is in.